bootstrap/sql/mysql/v004__create_db_connection_info.sql

```diff
@@ -15,3 +15,29 @@ CREATE TABLE IF NOT EXISTS thread_entity (
     INDEX (updatedAt)
 );
 
+CREATE TABLE IF NOT EXISTS glossary_entity (
+    id VARCHAR(36) GENERATED ALWAYS AS (json ->> '$.id') STORED NOT NULL,
+    name VARCHAR(256) GENERATED ALWAYS AS (json ->> '$.name') NOT NULL,
+    json JSON NOT NULL,
+    updatedAt BIGINT UNSIGNED GENERATED ALWAYS AS (json ->> '$.updatedAt') NOT NULL,
+    updatedBy VARCHAR(256) GENERATED ALWAYS AS (json ->> '$.updatedBy') NOT NULL,
+    deleted BOOLEAN GENERATED ALWAYS AS (JSON_EXTRACT(json, '$.deleted')),
+    timestamp BIGINT,
```

Comment:
`timestamp` sounds ambiguous

Should this be createdAt or something similar?

Reply:
That should not have been there... Removing it for now. We need to add createdAt at some point in time.